dub.json

```diff
@@ -7,8 +7,8 @@
   ],
   "license": "GPL-3.0",
   "dependencies": {
-    "dsymbol": ">=0.11.2 <0.12.0",
-    "libdparse": ">=0.15.4 <0.16.0",
+    "dsymbol": "~master",
```

Comment:
~master dependencies are not verifiable and may not update on builds. I have tagged a new dsymbol version which should be usable soon.

Reply:
> We should go ahead and just remove that line from the dub.json, and put Dsymbol's source directly in DCD/src/

Do you mean as a git submodule or just copied in place?